packages/reactive-element/src/reactive-element.ts

```diff
@@ -342,12 +337,8 @@ export const defaultConverter: ComplexAttributeConverter = {
         break;
       case Object:
       case Array:
-        // Do *not* generate exception when invalid JSON is set as elements
-        // don't normally complain on being mis-configured.
-        // TODO(sorvell): Do generate exception in *dev mode*.
         try {
-          // Assert to adhere to Bazel's "must type assert JSON parse" rule.
```

Comment:
Please don't delete comments

Reply:
I was written some code my side, and thought myself share with you, maybe want improve some parts based on code I wrote my way
Next time, will check everything before open a PR, also, won't change some names

Reply:
Also, main improvement is on `getPropertyDescriptor`